1-development/list_posts_by_rating/src/main/java/com/mixfar/posts/utils/response/Response.java

```diff
@@ -0,0 +1,35 @@
+package com.mixfar.posts.utils.response;
+
+import com.mixfar.posts.utils.enums.ErrorCode;
+
+import java.io.Serializable;
+
+public class Response implements Serializable {
```

Comment:
Why do you extend this class with ```DataResponse``` and ```ErrorResponse```, if both are equal?

Reply:
The Logic is the same but the attribute is naming differently. If we return a `DataResponse` the structure should be like this:
`{}